index.html

```diff
@@ -4,9 +4,49 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="main.css">
     <title>Document</title>
 </head>
 <body>
-    
+    <div id="yellowBackground">
+        <h1>Život u Splitu</h1>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur bibendum lectus non urna facilisis sollicitudin. 
+            Aliquam id metus quam. Cras pellentesque id mauris vel accumsan. Praesent pretium eros lacus, eu mattis libero porttitor ac.
+            Mauris sit amet accumsan tortor.</p>
+        <span>Građani</span>
```

Comment:
https://github.com/jkoprcina/SecondUIDevHomework/blob/d1ec95c343585ed8659000384e31b2a77a2d8ae8/index.html#L16-L23
These should be defined as buttons. Additionally, you can group them in their own `div` for a cleaner look of the code.